apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -430,7 +432,11 @@ const AppearanceView = ({
 const AppearanceViewWrapper = () => {
   const { data: user, isPending } = trpc.viewer.me.useQuery();
   const { isPending: isTeamPlanStatusLoading, hasPaidPlan } = useHasPaidPlan();
-
+  const session = useSession();
+  const router = useRouter();
+  if (!!session.data?.user?.org?.id) {
+    router.replace("/settings/organizations/profile");
+  }
```

Comment:
I think we should just hide the tab instead of doing this

Reply:
Hey Udit, I already hide this tab in the settings layout. We added it because if a user tries to access the page using the link, then we redirect them to the profile page.